### PR TITLE
[SQOW-305] PX | Move participation components from Admin

### PR DESCRIPTION
**Pull request by jmorales-hu**
## Summary
Currently we need from the owl team to replicate the Participation UI in web, to achieve this we decided to move some components to material-hu to be reusable in web and admin projects.

## Screenshots, GIFs or Videos
--

## Jira Card
https://humand.atlassian.net/browse/SQOW-305

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -49,3 +49,18 @@ export {
...
+  MenuItem as ActionsMenuItem,
+} from './components/peopleExperience/ActionsMenu';
+export { default as ValueIndicator } from './components/peopleExperience/ValueIndicator';
+export { default as ResultContainer } from './components/peopleExperience/ResultContainer';
+export { default as ListWithDivider } from './components/peopleExperience/ListWithDivider';
+export {
+  ParticipationItemSkeleton,
+  ParticipationExpandableItem,
+  ParticipationItem,
+} from './components/peopleExperience/Participation';
```
Comment (marianozicavo) [inline src/index.ts:65]: Acá podes exportar todo en un solo objeto de peopleExperience? Así no nos confundimos con los componentes compartidos

File: src/components/peopleExperience/Participation.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    <Stack sx={{ minWidth: '130px', alignItems: 'flex-end' }}>
+      <LinearProgressWithLabel
+        value={total ? partial / total : 0}
+        sx={{ flexDirection: 'row' }}
+      />
+      <Typography
+        variant="body2"
+        color="secondary"
+      >
+        {t('EMPLOYEES_RATIO', { partial, total })}
```
Comment (marianozicavo) [inline src/components/peopleExperience/Participation.tsx:73]: Esto funciona bien? Si es así buenisimo

File: src/index.ts (hunk 1 above)
Comment (jmorales-hu) [reply src/index.ts:65]: Dale, lo cambio

File: src/components/peopleExperience/Participation.tsx (hunk 2 above)
Comment (jmorales-hu) [reply src/components/peopleExperience/Participation.tsx]: Lo cambié para recibirlo por props

**Review by marianozicavo — APPROVED** [review]

**Pull request merged**